src/liveMarkbookCalculator.js

```diff
@@ -336,8 +340,13 @@ loadMarkbook = function (studentID, classID, termID, topicID, title, refresh, st
             $('#markbookTable').html(msg.d);
             $('#MarkbookDialog').dialog('option', 'height', 'auto').dialog('open');
             $('#markbookTable td[mrkTble!=\'1\']').addClass('tdAchievement');
-            makeMarkbookEditable();
+            const percentageColumnIsEnabled = settings.percentageColumn;
+            makeMarkbookEditable(percentageColumnIsEnabled);
```

Comment:
Just access the `settings.percentageColumn` directly, since it's in the global scope.
```suggestion
            makeMarkbookEditable();
```